gazelle/README.md

```diff
@@ -58,7 +58,9 @@ gazelle_python_manifest(
     modules_mapping = ":modules_map",
     # This is what we called our `pip_install` rule, where third-party
     # python libraries are loaded in BUILD files.
-    pip_deps_repository_name = "pip",
+    pip_repository_name = "pip",
+    # If using pip_parse incremental, switch the following to True,
```

Comment:
nit: users don't think of that name, they think of "pip_parse" - should probably say both